packages/firestore/src/local/indexeddb_query_cache.ts

```diff
@@ -97,36 +97,76 @@ export class IndexedDbQueryCache implements QueryCache {
     );
   }
 
+  private saveMetadata(
```

Comment:
I was gonna claim ownership of the term 'metadata' (but Mike already shut me down). Can you make this more query-specific (updateQueryMetadata) ?

Reply:
I think `metadata` is fine, since it's a private method referring specifically to the instance-scoped field called `metadata`. I don't think it lays claim to an sdk-wide notion of `metadata`.